azerothcore/azerothcore-wotlk
# Fix(DB/Vendor): Restock time for Undermine Clam Chowder

## Changes Proposed:
-  Decrease the timer (it was 3hs) according to wowhead comments.

## Issues Addressed:

- Closes #9530

## SOURCE:


## Tests Performed:

- Tested in-game
![image](https://user-images.githubusercontent.com/19627365/144941248-08c0faf2-0cd0-4bf8-851b-423782fa53b7.png)


## How to Test the Changes:


Visit Jabbey (.go c 23424) and buy its limited cooking recipe 16767
Wait for restock, it should be 15 minutes now.
